src/cljs/main/broadfcui/page/workspace/create.cljs

```diff
@@ -183,33 +178,3 @@
                     (if success?
                       (nav/go-to-path :workspace-summary {:namespace project :name name})
                       (swap! state assoc :server-error (get-parsed-response false))))})))})
-
```

Comment:
Removing the control to launch the Create Workspace dialog because we are removing the ability to fetch the billing projects.